scripts/submit_skims.sh

```diff
@@ -577,70 +577,70 @@ ZH_HToBB_ZToQQ_BR=`echo "0.69911*0.5824" | bc`
-    ["ST_tW_top_5f_inclusive"]="-n 20 -x 35.85 -q short"
-    ["ST_t-channel_antitop"]="-n 100 -x 80.95 -q short"
-    ["ST_t-channel_top"]="-n 400 -x 136.02 -q short"
+    ["TTToHadronic"]="-n 120 -x ${FullyHadXSec} -q short   --isTTlike"
+    ["TTTo2L2Nu"]="-n 3069 -x ${FullyLepXSec} -q short     --isTTlike"
+    ["TTToSemiLeptonic"]="-n 400 -x ${SemiLepXSec} -q long --isTTlike"
+
+    ["DYJetsToLL_M-50_TuneCP5_13TeV-amc"]="-n 600 -x 6077.22 -g ${STITCHING_ON} --DY 0 -q long --isDYlike" # inclusive NLO
+    ["DYJetsToLL_LHEFilterPtZ-0To50"]="-n 200    -x 1409.22 -g ${STITCHING_ON} --DY 0 -q long  --isDYlike"
+    ["DYJetsToLL_LHEFilterPtZ-50To100"]="-n 200  -x 377.12 -g ${STITCHING_ON} --DY 0 -q long   --isDYlike"
+    ["DYJetsToLL_LHEFilterPtZ-100To250"]="-n 200 -x 92.24   -g ${STITCHING_ON} --DY 0 -q long  --isDYlike"
+    ["DYJetsToLL_LHEFilterPtZ-250To400"]="-n 795 -x 3.512   -g ${STITCHING_ON} --DY 0 -q short --isDYlike" # some jobs are killed even with a single file (in the short queue)
+    ["DYJetsToLL_LHEFilterPtZ-400To650"]="-n 200 -x 0.4826  -g ${STITCHING_ON} --DY 0 -q long  --isDYlike"
+    ["DYJetsToLL_LHEFilterPtZ-650ToInf"]="-n 212 -x 0.04487 -g ${STITCHING_ON} --DY 0 -q short --isDYlike"
+    ["DYJetsToLL_0J"]="-n 200 -x 4867.28  -g ${STITCHING_ON} --DY 0 -q long --isDYlike"
+    ["DYJetsToLL_1J"]="-n 200 -x 902.95   -g ${STITCHING_ON} --DY 0 -q long --isDYlike"
+    ["DYJetsToLL_2J"]="-n 200 -x 342.96   -g ${STITCHING_ON} --DY 0 -q long --isDYlike"
+
+    ["WJetsToLNu_TuneCP5_13TeV-madgraph"]="-n 40 -x 48917.48 -y 1.213784 -z 70 -q short --isDYlike" # for 0 < HT < 70
+    ["WJetsToLNu_HT-70To100"]="-n 100 -x 1362 -y 1.213784 -q short --isDYlike"
+    ["WJetsToLNu_HT-100To200"]="-n 100 -x 1345 -y 1.213784 -q short --isDYlike"
+    ["WJetsToLNu_HT-200To400"]="-n 250 -x 359.7 -y 1.213784 -q short --isDYlike"
+    ["WJetsToLNu_HT-400To600"]="-n 50 -x 48.91 -y 1.213784 -q short --isDYlike"
+    ["WJetsToLNu_HT-600To800"]="-n 50 -x 12.05 -y 1.213784 -q short --isDYlike"
+    ["WJetsToLNu_HT-800To1200"]="-n 50 -x 5.501 -y 1.213784 -q short --isDYlike"
+    ["WJetsToLNu_HT-1200To2500"]="-n 50 -x 1.329 -y 1.213784 -q short --isDYlike"
+    ["WJetsToLNu_HT-2500ToInf"]="-n 50 -x 0.03216 -y 1.213784 -q short --isDYlike"
+
+    ["EWKWPlus2Jets_WToLNu"]="-n 20 -x 25.62 -q short --isDYlike"
+    ["EWKWMinus2Jets_WToLNu"]="-n 20 -x 20.25 -q short --isDYlike"
+    ["EWKZ2Jets_ZToLL"]="-n 19 -x 3.987 -q short --isDYlike"
+
+    ["ST_tW_antitop_5f_inclusive"]="-n 20 -x 35.85 -q short --isTTlike"
+    ["ST_tW_top_5f_inclusive"]="-n 20 -x 35.85 -q short --isTTlike"
+    ["ST_t-channel_antitop"]="-n 100 -x 80.95 -q short --isTTlike"
+    ["ST_t-channel_top"]="-n 400 -x 136.02 -q short --isTTlike"
 
     ["GluGluHToTauTau"]="-n 60 -x 48.68 -y 0.06272 -q short"
     ["VBFHToTauTau"]="-n 40 -x 3.766 -y 0.06272 -q short"
     ["WplusHToTauTau"]="-n 30 -x 0.831 -y 0.06272 -q short"
     ["WminusHToTauTau"]="-n 30 -x 0.527 -y 0.06272 -q short"
```

Comment:
From Tobias: shouldn't the PNet signal refer only to H->bb decays? Here you have H->tautau.

Reply:
Sorry, I didn't quite get you mean here...

Reply:
These processes have a $H\rightarrow \tau\tau$ decay. Since you do not add the option explicitly, I would naively say they are considered by your "isHHlike" option. You say above "we check if the fatjet is matched with a signal-like event, and in that case, we assign the central corrections", but shouldn't the fatjet be matched with the $H\rightarrow bb$ signal instead? Here it seems you are looking for a $H\rightarrow \tau\tau$ jet Or am I misunderstanding what you're doing?

Reply:
In each sample, including the cited H --> tautau, the current procedure identifies potential Z or Higgs bosons decaying into two b-quarks at gen level. It is then checked whether the identified fatjet matches the one found at the generator level. If this is the case, the central corrections are considered. Now, in the case of single Higgs decaying into tautau, it is likely that what is potentially identified as a fatjet will not correspond to an actual H or Z event in bb (therefore it will not be considered as isHHlike)... naively, I would say the topology will be more similar to a DY (gluons), but I wasn't sure and did not specify a topology...

Reply:
If you don't specify the topology and no match to $Z\rightarrow bb$ or $H\rightarrow bb$ happens, then [this condition](https://github.com/LLRCMS/KLUBAnalysis/blob/8f4f9abca895f2580197631b99ad57dd26ee09a9/test/skimNtuple_HHbtag.cpp#L5354) is not satisfied, and the SFs are not applied. I'm not expecting this to impact the result in any way, but it should be clarified, including in the AN.

Reply:
Exactly, in this case no correction is assigned. But I see your point, if we want to specify our best guess for each sample, then I would indicate them as 'DYlike'